### ISSUE TITLE
False negative VAR02 reports

### ISSUE THREAD

**Issue by MarcinGmurczyk** [MarketSquare/robotframework-robocop#1688]
Hi, using robocop, version 8.1.0 with code

```robot
*** Variables ***
${3MAR_DESCRIPTION}     Lorem ipsum 2

*** Test Cases ***
Log Global Variable
    Log To Console    ${3MAR_DESCRIPTION}
```

robot check will show

```bash
tests/test.robot:2:1 VAR02 Variable '${3MAR_DESCRIPTION}' is assigned but not used
   |
 1 | *** Variables ***
 2 | ${3MAR_DESCRIPTION}     Lorem ipsum 2
   | ^^^^^^^^^^^^^^^^^^^ VAR02
   |
```

Since variable is actually used in tests I believe robocop should not report them.
